### PR TITLE
Home page and routes

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
#### Added Landing page and routing
localTailor

### CODE REVIEW

File: src/frontend/components/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const Categories = () => {
+  return (
+    <div className="featured-categories flex-dir-col">
+      <h1>Featured Categories</h1>
+      <p className="text-medium">
+        Below are the trending featured products of Local Tailor
+      </p>
+      <div className="categories flex-dir-row">
+        <p className="categorie">
+          <a className="text-decoration-none" href="/products">
```
Comment (irajbhinde) [inline src/frontend/components/Categories/Categories.jsx:13]: You can replace anchor tags with Link now

**Review by yogesh- — APPROVED** [review]
- Good folder structure
- Good UI

File: src/frontend/pages/signin/Signin.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                type="text"
+                id="email"
+                placeholder="srujana@outlook.com"
+              />
+            </div>
+            <div className="field">
+              <label for="password">Password</label>
+              <input
+                className="input-field"
+                type="password"
```
Comment (irajbhinde) [inline src/frontend/pages/signin/Signin.jsx:26]: You can add a show password icon here & on click of that icon, change the input type from pass to text. It's mentioned in p0 requirements

**Comment by SRUJANA-PENUGONDA13** [comment]
Thank You @yogesh-  @irajbhinde

File: src/frontend/components/Categories/Categories.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            Men
+          </a>
+        </p>
+        <p className="categorie">
```
Comment (irajbhinde) [inline src/frontend/components/Categories/Categories.jsx:17]: You can avoid code repetition here, you can add category at categories.js Or products.js in db folder & then map them over the HTML code

**Review by vermasavita — COMMENTED** [review]
Good to go. But first, make the changes as raj suggested. Approving for now. 

Great work on UI  BTW:-)

**Pull request merged**